### PR TITLE
Capstone4 - Deliverable 2

### PR DESCRIPTION
**Pull request by giovponce**
## Description

< Capstone 4 - Second Deliverable >

- Product List Page
- Sidebar filters for mobile and desktop view
- Navigation from home to product list page
- Navigation to Home on header logo
- Pagination controls added (not activated yet)
- Loader for product grid
- Rendered data from mock files.

## Type of Change

- [x] New feature (non-breaking change which adds functionality)
- [x] Styles (Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc))

## What is the deliverable?
 Capstone 4 - Second Deliverable

## Link
https://oliviahomedecor.netlify.app/

### CODE REVIEW

File: src/Components/MobileSidebar.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  return (
+    <StyledMobileFilterContainer>
+      {categoriesSelection ? (
+        <>
+          <div>
+            {categories.results.map((category, index) => {
+              return (
+                <ChipMobile key={index} onClick={(e)=> {toggleCategorySelection(category.id, index)}}>
+                  <ChipMobileContent>
+                    <StyledMobileCheckbox id="input" type="checkbox" value={category.data.name} checked={checkedState[index]} onChange={e => {}}/>
```
Comment (sergio-vf) [inline src/Components/MobileSidebar.js]: you can remove the onchange if not needed

File: src/Pages/ProductList.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { StyledFlexCenteredContainer, BlockedStyledContainer } from '../utils/StyledComponents/Styled';
+import Sidebar from '../Components/Sidebar';
+import ProductGrid from '../Components/ProductsGrid';
+import Pagination from '../Components/Pagination';
+
+export default function ProductList() {
+
+  const [categories, setCategories] = useState([]);
```
Comment (sergio-vf) [inline src/Pages/ProductList.js:9]: Isnt it possible to handle both web and mobile in same variable? this way you can make your code cleaner.

**Review by sergio-vf — COMMENTED** [review]
Good job!!

File: src/Components/MobileSidebar.js (hunk 1 above)
Comment (giovponce) [reply src/Components/MobileSidebar.js]: removed :)

**Pull request merged**